### DISCUSSION TITLE
abpPermission

### DISCUSSION THREAD

**Discussion by zahra96389** [abpframework/abp#15382] [Q&A]
Hello 
How I can add abpPermission to Datagridview on this  [allowAdding]??

I want to assign abpPermission  to [allowAdding] attribute

thanks

**Comment by maliming** [comment]
hi

What do you mean `Datagridview`?
Can you show some code?

**Comment by zahra96389** [comment]
![image](https://user-images.githubusercontent.com/103983720/211823776-08fc2264-1a27-491e-806d-2453c2a5a440.png)

**Comment by maliming** [comment]
hi

https://docs.abp.io/en/abp/latest/UI/AspNetCore/JavaScript-API/Auth
https://docs.abp.io/en/abp/latest/UI/Angular/Permission-Management
